src/components/EnhancedTable/style.less

```
@@ -85,6 +85,11 @@
   width: 28px;
   height: 28px;
   padding: 0;
+  // Cap layout footprint at 24px (same as sort buttons / inline icons) so rows keep the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)